src/main/java/seedu/address/logic/commands/meeting/EditCommand.java

```diff
@@ -77,7 +77,16 @@ public CommandResult execute(ModelMeeting modelMeeting) throws CommandException
         }
 
         Meeting meetingToEdit = lastShownList.get(targetIndex.getZeroBased());
-        Meeting editedMeeting = createEditedMeeting(meetingToEdit, editMeetingDescriptor);
+        Meeting editedMeeting;
+
+        try {
+            editedMeeting = createEditedMeeting(meetingToEdit, editMeetingDescriptor);
+        } catch (IllegalArgumentException e) {
```

Comment:
where will IllegalArgumentException be thrown from?

Reply:
`createEditedMeeting()` throws IllegalArgumentException when it creates an invalid Meeting object through Meeting's constructor.